pkg/helm/helm.go

```diff
@@ -27,16 +28,25 @@ import (
 
 // Helm represents all current releases that we can find in the cluster
 type Helm struct {
-	CurrentReleases []*Release
-	Outputs         []*api.Output
-	Version         string
-	Kube            *kube
+	Releases []*Release
+	Outputs  []*api.Output
+	Version  string
+	Kube     *kube
 }
 
-// Release is a single version of a chart release
 type Release struct {
-	Name     string `json:"metadata.name" yaml:"metadata.name"`
-	Manifest string `json:"manifest,omitempty"`
+	Name     string `json:"name"`
+	Chart    *Chart `json:"chart"`
+	Manifest string `json:"manifest"`
+}
+
+type Chart struct {
```

Comment:
Since we only ever care about the metadata, maybe we should consider just using that? It might be easier to leave it this way to the Marshal is easier. Just a random thought

Reply:
I think having it like this is easier to reason with if we decide to add any other chart data at a later time. That's just my thought, I'm fine with either.